tests/io/vasp/test_outputs.py

```diff
@@ -1736,6 +1819,32 @@ def test_standard(self):
         finally:
             sys.stdout = saved_stdout
 
+    def test_vaspwave(self):
```

Comment:
This functions checks whether the information read from legacy and hdf5 files are same or not.